alignak/daemons/brokerdaemon.py

```diff
@@ -460,6 +460,8 @@ def setup_new_conf(self):  # pylint: disable=R0915,R0912
             else:
                 name = 'Unnamed broker'
             self.name = name
+            # Set my own process title
+            self.set_proctitle(self.name)
```

Comment:
This is managed in self.do_daemon_init_and_start() in main function. So if you add this here, it mean the new_conf modify the satellite name? but it's defined in the ini file, so I think we could not add this.